css/css-lists/parsing/list-style-image-computed.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Lists: getComputedValue().listStyleImage</title>
+<link rel="help" href="https://drafts.csswg.org/css-lists-3/#propdef-list-style-image">
+<meta name="assert" content="list-style-image computed value is as specified.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/computed-testcommon.js"></script>
+<style>
+  #target {
+    font-size: 40px;
+  }
+</style>
+</head>
+<body>
+<div id="target"></div>
+<script>
+test_computed_value('list-style-image', 'none');
+
+test_computed_value('list-style-image', 'url("https://example.com/")');
+
+test_computed_value('list-style-image', 'linear-gradient(to left bottom, red , blue )', 'linear-gradient(to left bottom, rgb(255, 0, 0), rgb(0, 0, 255))');
```

Comment:
Can you add a test with an existing symbolic value and ensuring it doesn't get serialized back?

Reply:
I don't follow.

Reply:
Sorry, this must be about list-style-type. I removed the list-style-type tests as they were reviewed in https://chromium-review.googlesource.com/c/chromium/src/+/1664093

I expected then to get merged automatically but https://github.com/web-platform-tests/wpt/pull/17495 hasn't merged them.  I'm not sure what the issue is.

Reply:
The list-style-type tests have now been merged in https://github.com/web-platform-tests/wpt/pull/17526